spec/dummy/config/routes.rb

```diff
@@ -6,7 +6,7 @@
 
   mount_graphql_devise_for(
     'Admin',
-    authenticable_type: Types::CustomAdminType,
+    authenticatable_type: Types::CustomAdminType,
     skip:               [:sign_up, :check_password_token],
     at:                 '/api/v1/admin/graphql_auth'
```

Comment:
This and the above need to be aligned too

Reply:
Done.